README.md

```diff
@@ -85,6 +85,57 @@ closeServices
 
 ```
 
+## RPC Events
+
+When exposing RPC methods a number of events are fired from the RPC client to assist with logging and debugging.
+
+```js
+var lokeIpc = require('loke-ipc');
+
+lokeIpc.connect('amqp://localhost')
+.then(function(connection) {
+  return connection.getRpcClient();
+})
+.then(function(client) {
+  client.on('request:start', doStuff);
+  client.on('request:complete', doStuff);
+  client.on('request:error', doStuff);
+});
+```
+
+### request:start
+
+```js
+client.on('request:start', function(e) {
+  console.log(e.method); // the method name eg "getCustomers" (string)
+  console.log(e.request); // the full request object
```

Comment:
hmm... i'll have to try and reverse engineer it... not documented anywhere at the moment